cpp/src/lance/io/reader.h

```diff
@@ -73,44 +87,54 @@ class FileReader {
   ::arrow::Result<std::shared_ptr<::arrow::Table>> ReadTable(
       const lance::format::Schema& schema) const;
 
+  /// Consolidate read by offset and read by indices parameters.
+  struct GetArrayParams {
```

Comment:
"GetArrayParams" sounds like a function that's retrieving some kind of array parameters. This is more like `ArrayTakeParams` or `ArrayReadParams`

Reply:
Fixed.